# Author: Caspar van Leeuwen, SURF
# This particular test seems sensitive to the behavior of the underlying BLAS library
# As such, it fails on ARM, see https://github.com/EESSI/software-layer/pull/916#issuecomment-2669244149
# The fix here is to pick a more stable starting point, that is less sensitive to the underlying BLAS behavior
# Also see this upstream issue: https://github.com/astropy/astropy/issues/15650
diff -Nru astropy-7.0.0-orig/astropy/modeling/tests/test_quantities_fitting.py astropy-7.0.0/astropy/modeling/tests/test_quantities_fitting.py
--- astropy-7.0.0-orig/astropy/modeling/tests/test_quantities_fitting.py	2025-03-04 23:32:36.606186987 +0100
+++ astropy-7.0.0/astropy/modeling/tests/test_quantities_fitting.py	2025-03-04 23:33:33.858927961 +0100
@@ -116,7 +116,7 @@
     x, y = _fake_gaussian_data()
 
     # Fit the data using a Gaussian with units
-    g_init = models.Gaussian1D(amplitude=1.0 * u.mJy, mean=3 * u.cm, stddev=2 * u.mm)
+    g_init = models.Gaussian1D(amplitude=1.0 * u.mJy, mean=4 * u.cm, stddev=3 * u.mm)
     g = fitter(g_init, x, y)
 
     # TODO: update actual numerical results once implemented, but these should
